rviz_common/src/rviz_common/screenshot_dialog.cpp

```diff
@@ -119,10 +121,11 @@ void ScreenshotDialog::onTimeout()
 
 void ScreenshotDialog::takeScreenshotNow()
 {
+  QScreen * screen = main_window_->windowHandle()->screen();
```

Comment:
Can `screen` or `windowHandle()` ever be `NULL`?

Reply:
They shouldn't be, no, but there was some case where it was happening. We couldn't get that from gdb, however, because it had incomplete type information to call the methods. But when we split it up into variables for each step, it wouldn't segfault :/